CosmosGettingStartedTutorial/CosmosGettingStarted.csproj

```diff
@@ -11,6 +11,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="*" />
+    <PackageReference Include="Microsoft.AspNetCore.JsonPatch" Version="6.0.0" />
```

Comment:
Why are these changes required? Can you try to remove these packages?

Reply:
added looking at the 'Update package references'. from documentation. after remove all those new packages added. the project is built successfully.